_i18n/en.yml

```diff
@@ -608,24 +610,40 @@ help_pages:
       or a mobile phone – at which you can receive security codes every time you sign
       in either by SMS or phone call.
     how-do-i-create-an-account-with-logingov: |-
-      To create a login.gov account, you will need: a valid email address, and a working phone number. We will also ask you to create a password.
-      #### Enter your email and confirm it
+      #### How do you create an account with login.gov?
+      <img src="site.baseurl/assets/img/help/1-login-gov-welcome.png" class="mt3 mb3 block" alt="Login.gov create account page help image" />
+      To create a login.gov account you’ll need a valid email address and a working phone number. We’ll also ask you to create a password.
 
-      Select “create an account” and enter your email address where indicated. Then check your email account for a message from us. Within the email is a link. Clicking the link will send you back to login.gov with your email confirmed.
-      #### Enter your password
+      Please be prepared to write down and store your password as well as the personal key that is generated for you in case you ever need to recover your account.
+
+      #### Enter your email and confirm it
+      <img src="site.baseurl/assets/img/help/2-login-gov-enteremail.png" class="mt3 mb3 block" alt="Enter your email help image" />
+      Select [“create an account”](https://secure.login.gov/) and enter your email address. Then check your email account for a message from us. Within the email is a link. Clicking the link will confirm your email address and send you back to login.gov.
```

Comment:
This is not the right URL for the create account page. It should be https://secure.login.gov/sign_up/enter_email